stubs/chevron/chevron/main.pyi

```diff
@@ -0,0 +1,10 @@
+from _typeshed import StrOrBytesPath
+from typing import Any
+
+from metadata import version as version
+from renderer import render as render
+
+_OpenFile = StrOrBytesPath | int
+
+def main(template: _OpenFile, data: _OpenFile | None = ..., **kwargs: dict[str, Any]) -> str: ...
```

Comment:
The way `**kwargs` are typed is confusing. If you write `**kwargs: int`, it means that you can do `the_function(foo=1, bar=2)`, and `kwargs` inside the function will be `{"foo": 1, "bar": 2}`. So if you write `**kwargs: dict[str, Any]`, it means you can do `main(..., yaml_loader={"string": something})`. But you're supposed to be able to pass at least strings, as in `main(..., yaml_loader="SafeLoader")`.

```suggestion
def main(template: _OpenFile, data: _OpenFile | None = ..., **kwargs: Any) -> str: ...
```